src/lib/eina/eina_inline_lock_posix.x

```diff
@@ -71,6 +71,10 @@
 #include <sys/types.h>
 #include <unistd.h>
 
+#ifndef _MSC_VER
```

Comment:
Same as in this [one](https://github.com/expertisesolutions/efl/pull/159/files#r449040104)